static/temp/foods.json

```diff
@@ -0,0 +1,259 @@
+    },
+    "overview": {
+      "coolFood": {
+        "icon": "IconName",
+        "text": "This is a cool food",
+        "moreInfo": {
+          "description": "Cool Food Meals are a simple and delicious way to help the planet.",
+          "linkText": "Learn More",
+          "link": "https://coolfood.org/consumer/"
+        }
+      },
+      "nutrientScore": "B",
+      "climateImpact": {
+        "title": "Climate Impact",
+        "emissions": "200kg CO2",
+        "emissionsUnit": "per serving",
+        "comparisons": [
+          {
+            "title": "0.2 gallons of diesel consumed",
+            "icon": "OilBarrelOutlinedIcon",
+            "order": 0
+          },
+          {
+            "title": "0.2 gallons of diesel consumed",
+            "icon": "DirectionsCarFilledOutlinedIcon",
+            "order": 1
+          }
+        ]
+      },
+      "tips": [
+        {
+          "name": "0.2 gallons of diesel consumed",
+          "icon": "OilBarrelOutlinedIcon",
+          "description": [
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            }
+          ]
+        },
+        {
+          "name": "0.2 gallons of diesel consumed",
+          "icon": "OilBarrelOutlinedIcon",
+          "description": [
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            },
+            {
+              "text": "Lorem ipsum dolor sit amen consectetur, adipisicing eliz. Harems alicia ab lula magna ea est autoed nescients ex architectonic rem!"
+            }
+          ]
+        }
+      ]
+    },
+    "images": ["/img1", "/img2", "/img3"],
+    "nutrition": {
+      "nutrientName": "Banana ripe, slightly ripe, raw",
+      "nutrientAmount": ["1 tablespoon", "1cup", "100g"],
+      "nutrientSource": "Source: USDA",
+      "nutrientGauges": [
+        { "name": "Protein", "percentage": "10%", "weight": "120g" },
+        { "name": "Carbs", "percentage": "20%", "weight": "120g" },
+        { "name": "Fats", "percentage": "30%", "weight": "120g" },
+        { "name": "Fiber", "percentage": "40%", "weight": "120g" }
+      ],
+      "nutrients": ["Protein", "carbohydrate", "water", "vitamin", "Fats", "Fiber", "Minerals"],
```

Comment:
Rename
1. `water` to `Water`
2. `vitamin` to `Vitamins`

Reply:
done